README.md

```diff
@@ -5,7 +5,7 @@ The application runs all the tests in the repository and outputs the results as
 
 ## Setting up the Environment
 
-This application requires Node v18 and makes use of the [Axios](https://axios-http.com/docs/intro) framework for HTTP request/response processing. [Node Download](https://nodejs.org/en/download)
+This application requires Node v20 and makes use of the [Axios](https://axios-http.com/docs/intro) framework for HTTP request/response processing. [Node Download](https://nodejs.org/en/download)
```

Comment:
Is this Node v20 a requirement? My understanding was that the Cozeva team wanted to make sure it worked with Node 18

Reply:
Ah too bad. I was not aware. True environment variables were added in 20. It make parameter handling so much easier. Who can we check with?

Reply:
I am going to double check this.

Reply:
@darkknight13 , any concerns with going to Node v20 here?

Reply:
I confirmed. The May vscode version 1.90 installed node 20 within the self contained electron update. I guess this is why I wouldn't have had a problem with the vscode launch.json settings for envFile.
[Node Env File](https://nodejs.org/en/learn/command-line/how-to-read-environment-variables-from-nodejs)
The --env-file command was added in 20.

I installed 18 and the --env-file command is not available, though you can still set the variables from the command line.

I will await @darkknight13 answer to decide a path forward. There are npm packages that do that same for older versions.